ksqldb-engine/src/test/java/io/confluent/ksql/function/udf/array/ArrayJoinTest.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.array;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
```

Comment:
nit: we prefer hamcrest asserts in our tests, you can look at other tests for examples of `assertThat(value, matches(matcher))`

Reply:
sure I can do that.
re-nit from my side: however, when it comes to consistency of the way things are done in existing UDFs we could discuss days if not weeks ;-)

Reply:
switched to hamcrest matchers